src/main/java/subsystems/Leds/LedConstants.java

```diff
@@ -0,0 +1,5 @@
+package subsystems.Leds;
```

Comment:
From the [documentation](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html) on package naming convention:

> Package names are written in all lower case to avoid conflict with the names of classes or interfaces.